scipy/interpolate/_rgi_cython.pyx

```diff
@@ -155,7 +152,7 @@ def find_indices(grid, bounds_error, xi):
         # index[index > grid[i].size - 2] = grid[i].size - 2
         # indices[i] = index
         for j in range(xi.shape[1]):
-            index = find_interval_descending(&grid_i[0], xi.shape[1], xi[i][j]) - 1
+            index = find_interval_ascending(&grid_i[0], xi.shape[1], xi[i][j]) - 1
```

Comment:
``find_interval_ascending`` takes an argument of ``const double *``, this is the neatest way I could massage the array into the right way, let me know if there's a better way to do it.